mypy/build.py

```diff
@@ -55,13 +55,15 @@ class BuildResult:
       files:   Dictionary from module name to related AST node.
       types:   Dictionary from parse tree node to its inferred type.
       errors:  List of error messages.
+      stale:   Set of stale modules that were rechecked
     """
 
     def __init__(self, manager: 'BuildManager') -> None:
         self.manager = manager
         self.files = manager.modules
         self.types = manager.type_checker.type_map
         self.errors = manager.errors.messages()
+        self.stale = manager.stale_modules
```

Comment:
Do you even need this? The check could just pull the info out of self.manager.stale_modules.